kgtk/cli/text_embedding.py

```diff
@@ -331,6 +333,8 @@ def add_arguments(parser: KGTKArgumentParser):
                         help="if set, will also save the embedding sentences to output.")
     parser.add_argument('-o', '--embedding-projector-metadata-path', action='store', dest='output_uri', default="",
                         help="output path for the metadata file, default will be current user's home directory")
+    parser.add_argument('--out-file', action='store', dest='output_file', default="",
```

Comment:
in other KGTK commands, `-o` is used for output file. Please make it consistent. I see `-o` is used for `--embedding-projector-metadata-path`, this needs to be updated